java-package/openbci_gui_helpers/src/main/java/openbci_gui_helpers/examples/TestDiscovery.java

```diff
@@ -1,19 +1,16 @@
 package openbci_gui_helpers.examples;
 
-import java.util.Map;
-
 import openbci_gui_helpers.GUIHelper;
 import openbci_gui_helpers.GanglionError;
+import openbci_gui_helpers.GUIHelper.GanglionDevice;
 
-public class TestDiscovery
-{
+public class TestDiscovery {
 
-    public static void main (String[] args) throws GanglionError
-    {
-        Map<String, String> map = GUIHelper.scan_for_ganglions (args[0], 3);
-        for (Map.Entry<String, String> entry : map.entrySet ())
-        {
-            System.out.println ("Key = " + entry.getKey () + ", Value = " + entry.getValue ());
+    public static void main(String[] args) throws GanglionError {
+        GanglionDevice[] devices = GUIHelper.scan_for_ganglions(args[0], 3);
+        for (GanglionDevice device : devices) {
+            System.out.println("Identifier = " + device.identifier + ", Mac Address = " + device.mac_address
```

Comment:
I take it this test is passing? Didn't see a shared test result.

Reply:
It passes for me, but it would be better if you also ran the test on your side so we can confirm it works on more than just my device.

Reply:
![image](https://github.com/OpenBCI/OpenBCI_GUI_Helpers/assets/84428015/24affbd1-d250-4c3c-b214-0d74c6311bd1)

Reply:
![image](https://github.com/OpenBCI/OpenBCI_GUI_Helpers/assets/84428015/eaaeb5ce-97b1-49c6-87d5-7e855772a932)